homeassistant/components/roborock/device.py

```diff
@@ -75,6 +78,7 @@ def __init__(
         self,
         unique_id: str,
         coordinator: RoborockDataUpdateCoordinator,
+        listener_request: list[RoborockDataProtocol] | None = None,
```

Comment:
```suggestion
        listener_request: list[RoborockDataProtocol] | RoborockDataProtocol | None = None,
```

So we can remove the ternary operator in select and sensor